admin/metabox/class-metabox-tab-section.php

```diff
@@ -8,38 +8,13 @@
 /**
  * Generates and displays the HTML for a metabox section.
  */
-class WPSEO_Metabox_Tab_Section implements WPSEO_Metabox_Section {
+class WPSEO_Abstract_Metabox_Tab_Section extends WPSEO_Abstract_Sectioned_Metabox_Tab {
```

Comment:
The naming of `WPSEO_Abstract_Sectioned_Metabox_Tab` is confusing to me. I suggest `WPSEO_Abstract_Metabox_Tab_With_Sections`.
What makes it confusing as well is that the actual class should not be abstract here.

```suggestion
class WPSEO_Metabox_Tab_Section extends WPSEO_Abstract_Metabox_Tab_With_Sections {
```